app-crypt/libsecret/libsecret-0.21.4.ebuild

```diff
@@ -0,0 +1,166 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit bash-completion-r1 gnome2 meson-multilib python-any-r1 vala virtualx
+
+DESCRIPTION="GObject library for accessing the freedesktop.org Secret Service API"
+HOMEPAGE="https://wiki.gnome.org/Projects/Libsecret"
+
+LICENSE="LGPL-2.1+ Apache-2.0" # Apache-2.0 license is used for tests only
+SLOT="0"
+
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="+crypt gnutls gtk-doc +introspection pam test test-rust tpm +vala"
+
+RESTRICT="!test? ( test )"
+REQUIRED_USE="
+	vala? ( introspection )
+	gtk-doc? ( crypt )
+	crypt? ( !gnutls )
+"
+
+DEPEND="
+	>=dev-libs/glib-2.44:2[${MULTILIB_USEDEP}]
+	crypt? ( >=dev-libs/libgcrypt-1.2.2:0=[${MULTILIB_USEDEP}] )
+	gnutls? ( >=net-libs/gnutls-3.8.2:0=[${MULTILIB_USEDEP}] )
+	pam? ( sys-libs/pam_wrapper )
```

Comment:
Huh, does it really need pam_wrapper at runtime? It's usually for tests..

Reply:
at least in the way pam/meson.build is written, they are not tunable by "test" option :/
```
Run-time dependency pam found: YES 1.6.1
Found CMake: /usr/bin/cmake (3.30.2)
WARNING: CMake Toolchain: Failed to determine CMake compilers state
Run-time dependency pam_wrapper found: NO (tried pkgconfig and cmake)

../libsecret-0.21.4/pam/meson.build:22:14: ERROR: Dependency "pam_wrapper" not found, tried pkgconfig and cmake
```

Reply:
I have just reported it to upstream:
https://gitlab.gnome.org/GNOME/libsecret/-/issues/96

Thanks for noticing